templates/zendesk.js

```diff
@@ -0,0 +1,29 @@
+window.zESettings = {
+  webWidget: {
+    authenticate: {
+      jwtFn: (/** @type{(token: string) => void} */ callback) => {
+        /** @type {string} */
+        const url = `${config.API_HOST}/auth/sso/zendesk`;
+        const token = window.sessionStorage.getItem('authToken');
+        fetch(url, {
+          method: 'POST',
+          headers: {
+            'x-tidepool-session-token': token
+          },
+          cache: 'no-cache',
+        }).then((response) => {
+          if (response.ok) {
+            const zendeskToken = response.headers.get('x-external-session-token');
+            console.info('Retrieve a new sso token for Zendesk');
+            return callback(zendeskToken);
+          }
+          console.warn('Zendesk callback SSO token: Missing token');
```

Comment:
No call to callback in this case ?

Reply:
I'm not sure what is best: not calling the callback seems to work well, at least it does not throw any error in the console. But if I call the callback with an invalid token (i.e. callback('invalid.token')) then it throws an exception...